### PR TITLE
Added Dashboard and learn page object model

### PR DESCRIPTION
**Pull request by naina-verma**
Added Dashboard and learn page object model

### CODE REVIEW

File: src/tests/work-item/work-item-list/page-objects/dashboard.page.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+let CommonPage = require('./common.page');
+let constants = require("../constants");
+let until = protractor.ExpectedConditions;
+
+class DashboardPage {
+
+ constructor(login) {
+ };
+ welcomeWrapper () {
+    return element(by.css("welcomeWrapper"));
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/dashboard.page.js]: Each of the UI elements that you define here should also have a "click" function - are they currently not click-able?

**Comment by joshuawilson** [comment]
[test]

File: src/tests/work-item/work-item-list/page-objects/dashboard.page.js (hunk 1 above)
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/dashboard.page.js]: Ah - OK - let's just leave it at that for now. Thx!!

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**